lib/utils/saveDataAndUploadToAzure.js

```diff
@@ -0,0 +1,19 @@
+const azureService = require('./azureService');
```

Comment:
Once the `azure-data-service` package is available this and the the `azureService` could be replaced with the AzureDataService class

Reply:
Yeah, I was thinking the same.